drivers/ethernet/eth_enc28j60.c

```diff
@@ -360,6 +493,16 @@ static int eth_enc28j60_init(struct device *dev)
 		return -EINVAL;
 	}
 
+#ifndef CONFIG_SPI_LEGACY_API
```

Comment:
@tbursztyka  ic checked that, and didn't work without this statement on HW, should this pin be configured in the SPI layer?